### ISSUE TITLE
check duplicate definition for 2 enum types

### ISSUE THREAD

**Issue by Nik-Sch** [vhdl-linter/vhdl-linter#404]
```vhdl
  type write_states is (IDLE, ARP_UPDATE);
  signal write_state : write_states;
  type read_states is (IDLE, ARP_READ_REQUEST, ARP_READ_DATA);
  signal read_state  : read_states;
```
should throw an error that `IDLE` is defined twice.

**Comment by g0t00** [comment]
Are you sure? ghdl takes this without complaining? You probably need to use some kind of selected name to select then write_states.IDLE or sth?

**Comment by Nik-Sch** [comment]
Yes, this is totally fine
